beast/observationmodel/ast/make_ast_xy_list.py

```diff
@@ -336,43 +366,54 @@ def pick_positions_from_map(
                     if catalog_boundary_radec:
                         within_bounds = catalog_boundary_radec.contains_points(
                             [[x, y]]
-                        )[0]  # N,2 array of AST X and Y positions
+                        )[
+                            0
```

Comment:
Is this line change necessary?

Reply:
Nope.  Formatting turned off for this line.